### DISCUSSION TITLE
월드 퍼블리시 시 ProjectSettings/Quality는 사용 불가한가요?

### DISCUSSION THREAD

**Discussion by ttoowa** [naverz/zepeto-studio-kor#848] [5. Q&A]
ProjectSettings/Quality 설정이 에디터에선 정상적용되나
QR빌드 및 퍼블리시시 설정이 무시되고 다르게 바뀝니다.

예전에는 세팅이 정상 적용되고 없던 현상이라
이것이 제페토 정책상 이유인지, 추후 해결될 이슈인지 궁금합니다.

https://github.com/naverz/zepeto-studio-kor/issues/751
여기에 동일한 내용의 이슈가 오픈되어있으나 아직까지는 답변이 없는 상태이고
해당설정 지원여부가 월드의 방향성에 영향을 미치기에
(그림자를 이용한 월드 등이 있을 수 있으나 현재 그림자가 표시되지 않음)
해결될 수 있는 이슈인지 여부를 답변을 해주시면 감사하겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요,  제페토 정책상 이유로 인해 기본적으로 월드의 퀄리티는 제페토 앱의 퀄리티 설정을 따르도록 되어 있습니다.
월드 개별적으로 퀄리티를 다르게 하는 추가 기능 개발 이슈이며 다음 마일스톤 목표로 진행 중입니다.
감사합니다.
